### DISCUSSION TITLE
Menu dessiner absent

### DISCUSSION THREAD

**Discussion by EEAM2023** [lairdubois/lairdubois-opencutlist-sketchup-extension#566] [General]
Menu dessiner absent, comment le faire apparaître? 
![thumbnail_Capture d’écran 2023-09-22 120302](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/146014540/407fa6e0-0e74-45b9-bea9-476b39cea53e)

**Comment by EEAM2023** [comment]
![thumbnail_Capture d’écran 2023-09-22 120237](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/146014540/43045170-1362-4006-8018-f1ad9213bd25)

**Comment by bbeaulant** [comment]
Bonjour,

Comme indiqué dans [la documentation](https://docs.opencutlist.org/features/parts/layout), le bouton "Dessiner" n'apparait que si : 
- tu utilises la version 2018 ou supérieure de SketchUp
- ton ordinateur est compatible avec la technologie WebGL.
